app/templates/about.jade

```diff
@@ -1,188 +1,363 @@
-extends /templates/base
-
 block content
-
-  img(src="/images/pages/about/coco_comic.jpg")
-
-  .row
-
-    .col-sm-6
-
-      h2(data-i18n="about.why_codecombat")
-        | Why CodeCombat?
-
-      p(data-i18n="about.why_paragraph_1")
-        | If you want to learn to program, you don't need lessons. 
-        | You need to write a lot of code and have a great time doing it.
-
-      p
-        span(data-i18n="about.why_paragraph_2_prefix")
-          | That's what programming is about. It's gotta be fun.
-          | Not fun like
-        span  
-        i(data-i18n="about.why_paragraph_2_italic")
-          | yay a badge
-        span  
-        span(data-i18n="about.why_paragraph_2_center")
-          | but fun like
-        span  
-        i(data-i18n="about.why_paragraph_2_italic_caps")
-          | NO MOM I HAVE TO FINISH THE LEVEL!
-        span  
-        span(data-i18n="about.why_paragraph_2_suffix")
-          | That's why CodeCombat is a multiplayer game,
-          | not a gamified lesson course. We won't stop 
-          | until you can't stop--but this time, that's a good thing.
-
-      p(data-i18n="about.why_paragraph_3")
-        | If you're going to get addicted to some game, 
-        | get addicted to this one and become one of the wizards of the tech age.
-
-      h2(data-i18n="about.press_title")
-        | Bloggers/Press
-
-      p
-        span.spr(data-i18n="about.press_paragraph_1_prefix")
-          | Want to write about us? Feel free to download and use all of the resources included in our
-        a(href="https://s3.amazonaws.com/CodeCombatMisc/press_packet.zip", data-i18n="about.press_paragraph_1_link") press packet
-        span(data-i18n="about.press_paragraph_1_suffix")
-          | . All logos and images may be used without contacting us directly.
-
-      h2(data-i18n="nav.contact")
-        | Contact
-      p
-        span CodeCombat Inc.
-        br
-        span 360 3rd St Suite 700 (Livefyre)
-        br
-        span San Francisco, CA 94107
-        br
-        a(href='mailto:team@codecombat.com') team@codecombat.com
-      if me.get('preferredLanguage') == 'pt-BR'
-        span CodeCombat, Brazil
-        br
-        span Rua 1814 Militão Chaves
-        br
-        span Natal, Brazil, RN 59064-500
-
-    ul.col-sm-6.team-column
-
-      ul.thumbnails
-
-        li.row
-
-          h2(data-i18n="about.team") Team
-
+  .style-flat(data-spy="scroll", data-target="#nav-container")
```

Comment:
Instead of having another wrapper element for the view, you can set the root element tag, id, classes and attributes in the view: http://backbonejs.org/#View-constructor